Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1170,8 +1176,74 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run:
                                                      '%Y-%m-%dT%H:%M:%S.000Z')
         latest_alert_id = alert_id
 
-    return incidents, \
-        {'last_fetched_alert_create_time': latest_alert_create_date, 'last_fetched_alert_id': latest_alert_id}
+    res = {'last_fetched_alert_create_time': latest_alert_create_date, 'last_fetched_alert_id': latest_alert_id}
+    return incidents, res
+
+
+def process_search_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Gets arguments for a process search task, and returns the task's id and status.
+    """
+    process_name = args.get('process_name', '')
+    process_hash = args.get('process_hash', '')
+    event_id = args.get('event_id', '')
+    query = args.get('query', '')
+    limit = args.get('limit')
+    if not limit:
+        limit = 20
```

Comment:
```suggestion
    limit = args.get('limit', '20')
```

Reply:
It fails the playbook as the test puts limit="" which exists but not empty so we get "" instead of 20.

Reply:
Ok then.